File: solutions/python/raindrops/1/raindrops.py
def convert(number): #bazinga
    output = ""
    if number % 3 == 0:
        output +=  "Pling"
    if number % 5 == 0:
        output += "Plang"
    if number % 7 == 0:
        output += "Plong"
    elif (number % 3 != 0 and  number % 5 != 0 and number % 7 != 0):
        output = str(number)
    return output

File: solutions/go/raindrops/1/raindrops.go
package raindrops // Finished niave solution

import "fmt"

var dropNums = []int{3, 5, 7}
var dropWords = []string{"Pling", "Plang", "Plong"}

func Convert(number int) string {
	var rainDrop string
	var foundButADrop bool = false

	for i, v := range dropNums {
		//fmt.Println(k, v)
		if number%v == 0 {
			rainDrop = rainDrop + dropWords[i]
			foundButADrop = true
		}
	}

	if !foundButADrop {
		return fmt.Sprint(number)
	}

	return rainDrop
	//return "This is a test"
}
